src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -264,6 +265,43 @@ def _check_tbrom_points_generation_args(self, rom_name: str, namedselection: str
                 raise self._raise_error(msg)
         return True
 
+    def _check_tbrom_mesh_projection_args(self, rom_name: str, mesh: pv.DataSet, namedselection: str = None):
+        """
+        Check if the arguments of mesh projection method are valid. Raise a ``TwinModelError`` message if not.
+        """
+        tbrom = None
+        if self._check_rom_name_is_valid(rom_name):
+            tbrom = self._tbroms[rom_name]
+
+        if not tbrom._hasoutmcs:
+            msg = self._error_msg_for_rom_output_connection(rom_name)
+            raise self._raise_error(msg)
+
+        if mesh.n_cells == 0 and mesh.n_points == 0:
+            msg = self._error_msg_for_pv_mesh_empty(rom_name, mesh)
+            raise self._raise_error(msg)
+
+        if namedselection is not None:
+            if namedselection not in tbrom.named_selections:
+                msg = self._error_msg_for_unknown_named_selection(namedselection, tbrom)
+                raise self._raise_error(msg)
+        return True
+
+    def _check_tbrom_mesh_update_args(self, rom_name: str):
```

Comment:
`_check_tbrom_mesh_update_args` seems to be unused. Please remove-it if that is the case.

Reply:
in which case `_error_msg_for_tbrom_mesh_empty` will be useless as well.

Reply:
correct - changes done